x/log_writer.go

```diff
@@ -41,7 +39,9 @@ const (
 	backupTimeFormat = "2006-01-02T15-04-05.000"
 	bufferSize       = 256 * 1024
 	flushInterval    = 10 * time.Second
-	VerificationText = "Hello World"
+	//  old logs before https://github.com/dgraph-io/dgraph/pull/8323 contain deprecated verification text in header
+	VerificationTextDeprecated = "Hello World"
```

Comment:
would it be more useful to version them or something? What if we need to change this tomorrow again? We can also just do it the next time if that makes more sense.

Reply:
I hope we never have to look at this ever again :)  But in all seriousness I don't think we need to version it at this point.  If we do get in that position tomorrow we can just change these variable names.